doc/modules/sorters.rst

```diff
@@ -222,6 +222,34 @@ by saving the in-memory  :code:`test_recording` object to a binary file in the
 :code:`test-docker-folder` folder.
 
 
+What version of SpikeInterface is run in the container?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The spike-sorter specific images do NOT include the :code:`spikeinterface` package.
+This is done because the spike sorters are "frozen" to a specific version, while the :code:`spikeinterface` package
+is in constant evolution with new releases.
+
+When starting a container, the first step is then to install :code:`spikeinterface` and its dependencies.
+
+
+What version of :code:`spikeinterface` is installed? It depends!
+
+There are three options:
+
+1. **released PyPi version**: if you installed :code:`spikeinterface` with :code:`pip install spikeinterface`,
+   the latest released version will be installed in the container.
+
+2. **development :code:`main` version**: if you installed :code:`spikeinterface` from source from the cloned repo
+   (with :code:`pip install .`) or with :code:`pip install git+https://github.com/SpikeInterface/spikeinterface.git`,
+   the current development version from the :code:`main` branch will be installed in the container.
+
+3. **local copy**: if you installed :code:`spikeinterface` from source and you have some changes in your branch or fork
+   that are not in the :code:`main` branch, you can install a copy of your :code:`spikeinterface` packahe in the container.
+   To do so, you need to set en environment variable :code:`SPIKEINTERFACE_DEV_PATH` to the location where you cloned the
+   :code:`spikeinterface` repo (e.g. on Linux: :code:`export SPIKEINTERFACE_DEV_PATH="path-to-spikeinterface-clone"`.
+
+
```

Comment:
Maybe we could a receipe on how to build a new docker image with a specific SI version / commit.